pkg/components/page/page.go

```diff
@@ -48,42 +50,47 @@ func (p *page) Render(provider core.Provider, cell entity.Cell) {
 	}
 }
 
+// SetConfig sets the page configuration.
 func (p *page) SetConfig(config *entity.Config) {
 	p.config = config
 	for _, row := range p.rows {
 		row.SetConfig(config)
 	}
 }
 
+// SetNumber sets the page number and total.
 func (p *page) SetNumber(number int, total int) {
 	p.number = number
 	p.total = total
 }
 
+// GetNumber returns the page number.
 func (p *page) GetNumber() int {
 	return p.number
 }
 
+// Add adds a row to the page.
```

Comment:
// Add adds one or more rows to the page.

Reply:
done